k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -3517,7 +3527,7 @@ public void deleteDraft(final Account account, long id) {
             if (uid != null) {
                 LocalMessage message = localFolder.getMessage(uid);
                 if (message != null) {
-                    deleteMessages(Collections.singletonList(message), null);
+                    deleteMessages(Collections.singletonList(message.makeMessageReference()), null);
```

Comment:
Instead of creating a `MessageReference` just to satisfy the method signature please extract the code that does the actual work (using a `LocalMessage`) to a new method.